ext/standard/basic_functions.stub.php

```diff
@@ -1503,6 +1503,16 @@ class AssertionError extends Error
 {
 }
 
+#ifdef PHP_CAN_SUPPORT_PROC_OPEN
+/**
+ * @strict-properties
+ * @not-serializable
+ */
+final class Process
```

Comment:
Reserving the name "Process" may break some user code.
It may make sense to introduce and use some extension specific namespace(s).
e.g. ``standard`` or ``php\standard`` or ``php\ext\standard``

Reply:
I agree that it might be better to put this to the separate namespace. The only one that is allowed would be Standard.

Reply:
Thanks for the reminder, I'll put it into the `Standard` namespace